lib/OrdinaryDiffEqLowOrderRK/src/alg_utils.jl

```diff
@@ -0,0 +1,49 @@
+alg_order(alg::Euler) = 1
+alg_order(alg::SplitEuler) = 1
+alg_order(alg::Heun) = 2
+alg_order(alg::Ralston) = 2
+alg_order(alg::Midpoint) = 2
+alg_order(alg::BS3) = 3
+alg_order(alg::OwrenZen3) = 3
+alg_order(alg::BS5) = 5
+alg_order(alg::OwrenZen4) = 4
+alg_order(alg::OwrenZen5) = 5
+alg_order(alg::Tsit5) = 5
```

Comment:
We will want to make Tsit5 be its own set.

Reply:
Ok so for this I will make a separate OrdinaryDiffEqTsit inside lib

Reply:
OrdinaryDiffEqTsit5. Since it's the most common method and it's commonly used alone, making it as lean as possible is a virtue. Thanks.